imgee/__init__.py

```diff
@@ -22,20 +22,21 @@
 from .models import db
 from .tasks import TaskRegistry
 
-registry = TaskRegistry(os.getenv('ENV', 'production'))
-
-
-def mkdir_p(dirname):
-    if not os.path.exists(dirname):
-        os.makedirs(dirname)
-
+registry = TaskRegistry()
 
 # Configure the app
 coaster.app.init_app(app)
 migrate = Migrate(app, db)
 baseframe.init_app(app, requires=['baseframe', 'picturefill', 'imgee'])
 lastuser.init_app(app)
 lastuser.init_usermanager(UserManager(db, models.User))
+registry.init_app()
+
+# PYTHONPATH is `pwd` when testing and empty otherwise
+# using it to determine the project root
+# either get it from environment variable, or it's one level up from this init file
+app.project_root = os.environ.get('PYTHONPATH', '') or os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+print app.project_root
```

Comment:
Remove these two lines. They should be in the test script, not in code that runs in production.